### PR TITLE
Remove remaining xmlquery calls from build-namelist scripts

### PR DESCRIPTION
**Pull request by jgfouca**
[BFB]

### CODE REVIEW

File: components/mpas-ocean/cime_config/buildnml [hunk 1]
```diff
@@ -35,9 +35,11 @@ def buildnml(case, caseroot, compname):
     ocn_co2_type      = case.get_value("OCN_CO2_TYPE")
     atm_co2_const_val = case.get_value("CCSM_CO2_PPMV")
     ice_bgc           = case.get_value("MPASI_BGC")
-    nthrds_ocn        = case.get_value("NTHRDS_OCN")
     ocn_pio_typename  = case.get_value("OCN_PIO_TYPENAME")
     ninst_ocn         = 1 # Change if you want multiple instances... though this isn't coded yet.
+    ninst_ocn_real    = case.get_value("NINST_OCN")
```
Comment (jonbob) [inline components/mpas-ocean/cime_config/buildnml]: Is there a reason for this to have the _real suffix? More curious than anything....

Comment (jgfouca) [reply components/mpas-ocean/cime_config/buildnml]: If you look at the line above, ninst_ocn is hardcoded to 1. The build-namelist scripts were using xmlquery to get the real NINST_OCN (not hardcoded to 1).

Comment (jgfouca) [reply components/mpas-ocean/cime_config/buildnml]: This is probably a flaw in the mpas scripts, with buildnml hardcoding ninst to 1 and build-namelists using the real NINST value, but I didn't want to change behavior.

Comment (jonbob) [reply components/mpas-ocean/cime_config/buildnml]: Got it -- thanks.

**Comment by jgfouca** [comment]
Merged to next.

**Pull request merged**